content/guides/embed/box-view/create-preview.md

```diff
@@ -0,0 +1,137 @@
+---
+rank: 40
+related_guides:
+  - embed/box-view/create-preview
+required_guides:
+  - embed/box-view/setup
+  - embed/box-view/upload-file
+  - embed/ui-elements/installation
+  - embed/ui-elements/preview
+alias_paths: []
+---
+
+# Creating a File Preview
+
+Once a file has been uploaded to the app token application, it may be previewed
+using two different methods:
+
+* Direct embed: A simple HTML iframe component with a custom embed link.
+* Customized previewer: A fully customized preview widget using Box UI Elements.
+
+## Direct Embed (iframe)
+
+The direct embed link is a light-weight API method that provides limited
+options to customize the preview experience in your application. Due to the
+light-weight nature of the method, the API doesn't give you options to respond
+to client-side actions such as scrolling in the case of documents, play/pause
+interactivity for videos, rotating an image, etc.
+
+There are two steps towards creating a direct iframe embed for Box View:
+
+1. Generate an embed URL for the file
+2. Add the embed URL to an iframe
+
+### Generate an embed URL for the file
+
+To create a public file preview URL for a file that was uploaded to the app
+token application, you may either use a direct SDK method or make the request
+directly to the APIs.
+
+<Message type='notice'>
+  When generating the embed URL directly from the APIs, use the
+  [Get File Information endpoint](endpoint://get_files_id) and request
+  `expiring_embed_link` via the `fields` parameter.
+</Message>
+
+<Tabs>
+  <Tab title='cURL'>
+    curl https://api.box.com/2.0/files/FILE_ID?fields=expiring_embed_link \
```

Comment:
FYI: code blocks still need to be wrapped into code blocks. The Tab element doesn't do that as-is.